blocks/cards/cards.css

```diff
@@ -0,0 +1,81 @@
+.block.cards {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  grid-auto-rows: 1fr;
+  gap: 1rem;
+}
+
+.block.cards .card .url {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  padding: 0;
+  opacity: 0;
+  margin: 0;
+}
+
+.block.cards .card .url > a {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.block.cards .card {
+  position: relative;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+  padding: 0.75rem;
+  font-size: 1rem;
+  border-radius: 16px;
+}
+
+.block.cards .card:hover {
+  box-shadow: 0 0 31px -8px rgb(10 10 10 / 30%);
+}
+
+.block.cards .card .title {
+  align-self: stretch;
+  font-weight: 700;
+
+  /* max-width: 80%; */
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.block.cards .card img {
+  width: 42px;
+  height: 42px;
+}
+
+@media (width > 900px) {
+  .block.cards .card {
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    text-align: left;
```

Comment:
Not just the text, we want the content to be left aligned.
I think `justify-content: flex-start` should help fix it. can you try that?

Reply:
Oh ok I misunderstood I though you were talking about when it wraps the text. Will change that, also on the piramal website they are doing something weird: on some viewports the title is converted in 2 different paragraphs of length 1/2 words. On other viewports they just insert <br> tags and on others they dont do anything. Should I do something about it or just ignore it as it's too much overhead?

Reply:
I think lets not break that into separate words. We can have just the `word wrap` in place so when the viewport width reduces, the text wraps properly as per the expected width.